src/commands/currency/transfer.js

```diff
@@ -33,11 +33,12 @@ module.exports = class Economy extends Command {
    const recipient = await mongoCurrency.findUser(member.id, message.guild.id);
    const amt = args.amount;
 
-   if (user.coinsInWallet > amt){
+   if (user.coinsInWallet >= amt){
       await mongoCurrency.giveCoins(member.id, message.guild.id, amt );
+      await mongoCurrency.takeCoins(message.member.id, message.guild.id, amt);
```

Comment:
take coin is not a function in current package!

Reply:
ah yes, it is deductCoins